src/ClientHandle.cpp

```diff
@@ -2082,6 +2141,15 @@ void cClientHandle::SendBlockChanges(int a_ChunkX, int a_ChunkZ, const sSetBlock
 
 
 
+void cClientHandle::SendCameraSetTo(const cEntity & a_Entity)
```

Comment:
This are new packets... Unless you mix them with something else, but I have no idea about that...